build.zig

```diff
@@ -1,71 +1,50 @@
 const std = @import("std");
 const Pkg = std.build.Pkg;
 
+const this_dir = thisDir();
```

Comment:
Since comptime functions are effectively memoized by Zig, there's no benefit in having this be a variable FWIW. I'm okay with it, though.